### ISSUE TITLE
Touching message renders black screen

### ISSUE THREAD

**Issue by jomo** [Marlinski/Rumble#7]
I touched the right quarter of a message I just sent. The app goes entirely black. I also noticed the message body has a height of ~5 lines where only one would be required. I'm not sure if that's related or not.

When I touch the header or any of the left 3/4 part, it shows the message on its own. This is also the case for the "pre-installed" messages, but only when touching the message header. Touching the body on those messages doesn't do anything.

I'm using Android 5.1.1 with CyanogenMod 12.1

**Comment by Marlinski** [comment]
Yeah that's not a bug (as in unintented behaviour) but I have to work on the UI. The idea was to be able to show a status on its own just like in other twitter client. but as you noticed there is also several clickable area already (the hashtags in the text, the menu top right, the avatar) so what's left are the empty area like the header or text area without text. 

Anyway thanks for your feedback, I'll work on it to make it more user-friendly.
